README.md

```diff
@@ -179,6 +179,16 @@ Added in 3.3.0.
 
 Defaults to `false`.
 
+### savePicsCache
+
+If `enablePicsCache` is enabled, save product info to disk. (TODO: Improve description)
+
+Added in 4.24.0
+
+Defaults to `false`.
+
+**Warning:** This will significantly increase the storage space that is needed!
```

Comment:
How much disk space does this need? Might it be worthwhile to gzip the json before we store it?

Reply:
I should add "potentially", because it all depends on how many products you will be requesting. But it will build up over time, of course.
As for your first question, I once saved the entire SteamUser instance to disk as JSON. This was around 140 MB and the majority of that data consisted of the pics cache JSON. That was 4 years ago and my library has grown from then. Anyway, it will be in the order of hundreds of MB. Maybe getting into the gigabytes eventually, if you are going to monitor all pics changes. Using the custom file engine, one may implement a database hook instead.

Reply:
So, with the main functionality added now, I got this by just saving my owned products infos to disk: 
![image](https://user-images.githubusercontent.com/4411977/169673733-d7636e6a-919e-4adb-9158-b6c8a26d0f5f.png)

I think that's reasonable, especially with my unreasonable library size :P
People can always choose to not save pics cache to disk anyway.
The danger lies if people also use `picsCacheAll`, and potentially save product infos of  everything. I've added a warning especially for this.